patterns/lcesave.hexpat

```diff
@@ -0,0 +1,156 @@
+#pragma author DexrnZacAttack
+#pragma description Pattern for Minecraft LCE Save Files (after decompression)
+#pragma version 1.2.0
+#pragma disclaimer This pattern is not affiliated with Mojang Studios nor Microsoft. Minecraft is a trademark of Mojang Studios and Microsoft.
+#pragma endian big
+/*
+ * ^ switch this to little for Switch, PS4, or Xbox One
+ * so you'll probs have to switch all of the be to le
+*/
+
+// NOTE: This pattern was only tested on Nightly@974c4ba, things may break/not work on older versions.
+
+#include <std/string.pat>
```

Comment:
I believe that this line isn't needed in v1.35.4 of ImHex. The `import` line below should do just fine.

Reply:
> I believe that this line isn't needed in v1.35.4 of ImHex. The `import` line below should do just fine.

Alright I'll probably fix that in my hexpat later, right now I'm trying to find a good file for the example folder and fix some issues I had with the NBT library (TypeTag not found iirc)